dora/core/common/src/main/java/alluxio/membership/ServiceEntity.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.membership;
+
+import io.etcd.jetcd.support.CloseableClient;
+
+import java.io.Closeable;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.ReentrantLock;
+
+/**
+ * Base Entity class including information to register to Etcd
+ * when using EtcdMembershipManager.
+ */
+public class ServiceEntity implements Closeable {
+  private CloseableClient mKeepAliveClient;
+  AlluxioEtcdClient.Lease mLease; // used for keep alive(heartbeating) will not be set on start up
```

Comment:
getter/setter?

Reply:
this is intended to be visible/manipulatable within package only